Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -919,11 +924,60 @@ def change_attack_pattern_to_stix_attack_pattern(indicator: dict[str, Any]):
 
         return indicator
 
+    @staticmethod
+    def get_entity_b_type_and_value(related_obj: str, id_to_object: dict[str, dict[str, Any]],
```

Comment:
Isnt "get_indicator_type_and_value" a better name? It also matches your description (As its not limited to entity_b)